src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultIconInput/styles.ts

```diff
@@ -0,0 +1,48 @@
+import styled from '@emotion/styled';
+import {flexibleRowDiv} from '../../shared';
+import {GreyBackground, IgnitusBlue, Red} from '../colors';
+
+import Icon from '../../../ignitus-Utilities/Components/icon';
+
+
+export const InputContainer = styled(flexibleRowDiv)`
+  border: none;
+  border-radius: 8px;
+  padding: 0.2rem;
+  position: absolute;
+  width: 436px;
+  height: 50px;
+  background: ${GreyBackground};
+  &:focus {
+    color: ${Red};
+  }
+`;
+
+export const Input = styled.div`
```

Comment:
please don't declare input as div semantically it should be input html element, I have corrected this locally as of now but keep in mind for future.